apiclient/types/toolreference.go

```diff
@@ -25,7 +25,7 @@ type ToolReference struct {
 	Error       string            `json:"error,omitempty"`
 	Builtin     bool              `json:"builtin,omitempty"`
 	Description string            `json:"description,omitempty"`
-	Credentials []string          `json:"credential,omitempty"`
+	Credentials []string          `json:"credentials,omitempty"`
```

Comment:
was this causing problems before?

Reply:
No, it wasn't causing any problems, just figured I should fix it since I saw it (and I created it).